book/src/security.md

```diff
@@ -0,0 +1,12 @@
+Security
+========
```

Comment:
I think we need a header `#` here and a blank line after the header to make Markdown linter happy

```suggestion
# Security

========
```